source/rst/tax_smoothing_1.rst

```diff
@@ -501,44 +509,44 @@ spending process.
     plt.ylabel('Taxation')
     plt.show()
 
-Python class to solve Markov jump linear quadratic control problems
--------------------------------------------------------------------
+Python Class to Solve Markov Jump Linear Quadratic Control Problems
+===================================================================
 
 To implement the extension to the Barro model in which :math:`p_{t,t+1}`
 varies over time, we must allow the M matrix to be time-varying. From
 the mapping of the Barro model into the LQ framework, this means that
-our Q and W matrices will now also vary over time. We can solve such a
+our :math:`Q` and :math:`W` matrices will now also vary over time. We can solve such a
 model using the LQ_Markov class, which solves Markov jump linear
-quandratic control problems as described above.
+quandratic control problems as described above
 
 The code for the class can be viewed
-`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__.
+`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__
 
 The class takes a variable number of arguments, to allow for there to be
 an arbitrary :math:`N` states of the world. To accomodate this, the
 matrices for each state of the world must be held in a “namedtuple”. The
 value and policy functions are then found by iterating on the system of
 algebraic matrix Riccati equations. The solutions for :math:`P,F,\rho`
-are stored in Python “dictionaries”.
+are stored in Python “dictionaries”
```

Comment:
could you go through these paragraphs and delete the full stops and put sentences on separate lines